sootty/save.py

```diff
@@ -0,0 +1,66 @@
+import os
+import subprocess
+
+
+def save_query(save, name, wires, br, length, start, end, display):
+    save = os.getenv("HOME") + "/.config/sootty/save/" + save
+    if is_save_file(save) is True:
+        val = input(
```

Comment:
Yeah so I'll give it a default filename in case nothing provided with the -S flag, and that should do it I think?

Reply:
Nvm, I understood it wrong, but I thought this might be better from a user perspective, mostly because I've often erased files I didn't mean to by mistake